WarmUp/Program.cs

```diff
@@ -5,6 +5,7 @@
 using System.Net.Http;
 using System.Threading;
 using System.Threading.Tasks;
+using GenderPayGap.Extensions;

```

Comment:
The WarmUp script is its own, entirely separate application.
We run it on the build server, not in a context that has the configuration necessary to change the VirtualDateTime.
So, for WarmUp, I think we shouldn't use VirtualDateTime.

Reply:
Good point, I've reverted these changes and force pushed